src/helpers/tests/geoChart/data.spec.ts

```diff
@@ -2,7 +2,19 @@
 import { Execution, VisualizationObject } from "@gooddata/typings";
 import { COLOR_ITEM, LOCATION_ITEM, SEGMENT_BY_ITEM, SIZE_ITEM, TOOLTIP_TEXT_ITEM } from "./fixtures";
 import { getExecutionResponse, getExecutionResult } from "../../../../stories/data/geoChart";
-import { getGeoData } from "../../geoChart/data";
+import { getGeoData, getLocation } from "../../geoChart/data";
+
+describe("getLocation", () => {
+    it("should return [lng, lat] from location string", () => {
+        const location = getLocation("44.500000;-89.500000");
+        expect(location).toEqual([-89.5, 44.5]);
+    });
+
+    it.each(["", "123"])("should return null when input is %s", (input: string) => {
```

Comment:
Please make sure it prints `should return null when input is '%s'` so that in console log we can see it as string

Reply:
good catching. 
Thank you